code/modules/mob/living/carbon/life.dm

```diff
@@ -53,9 +53,13 @@
 
 	var/datum/gas_mixture/breath
 
-	if(health <= HEALTH_THRESHOLD_CRIT || (pulledby && pulledby.grab_state >= GRAB_KILL && !getorganslot("breathing_tube")))
+	if(health <= HEALTH_THRESHOLD_FULLCRIT || (pulledby && pulledby.grab_state >= GRAB_KILL && !getorganslot("breathing_tube")))
 		losebreath++
 
+	else if(health <= HEALTH_THRESHOLD_CRIT)
+		if(prob(33))

```

Comment:
if you stay still, you can't die from being crit alone

Reply:
?

You take damage on average once ever 3 ticks rather than once per tick.

Reply:
you regenerate oxygen damage faster than you'll take it, since crit's damage is entirely losebreath based

Reply:
Why even leave it to prob instead of a flat reduction?

Reply:
losebreath doesn't do decimals

Reply:
I will fix this